programs/address-lookup-table-tests/tests/common.rs

```diff
@@ -0,0 +1,107 @@
+use {
+    solana_address_lookup_table_program::{
+        id,
+        processor::process_instruction,
+        state::{AddressLookupTable, LookupTableMeta},
+    },
+    solana_program_test::*,
+    solana_sdk::{
+        account::AccountSharedData,
+        clock::Slot,
+        hash::Hash,
+        instruction::Instruction,
+        instruction::InstructionError,
+        pubkey::Pubkey,
+        signature::{Keypair, Signer},
+        slot_hashes::SlotHashes,
+        transaction::{Transaction, TransactionError},
+    },
+    std::borrow::Cow,
+};
+
+#[allow(dead_code)]
```

Comment:
it's so annoying to have to put these everywhere :disappointed:

Reply:
Changed to a module-wide allow, which is slightly less annoying haha